CRM/Cdntaxreceipts/Task/IssueAggregateTaxReceipts.php

```diff
@@ -370,7 +370,7 @@ function postProcess() {
             //CRM-920: Thank-you Email Tool
             if ($sendThankYouEmail) {
               $thankyou_html = $this->getThankYouHTML([$contribution->id], $from_email_address, $params);
-              if ($thankyou_html != NULL)
+              if ($thankyou_html != NULL && !$previewMode)
```

Comment:
Why not do this directly when we set `$sendThankYouEmail` (line 251), as this is the purpose of that variable: to define whether or not we should send a thank you email...